### PR TITLE
Initial implementation of the Swipe page

### PR DESCRIPTION
**Pull request by joshcampitelli**

### CODE REVIEW

File: components/Restaurant.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+import Indicator from './Indicator';
+import RestaurantDetails from './RestaurantDetails'
+
+export default function Restaurant(props) {
+    const [count, setCount] = useState(0);
+    let images = props.images;
+    let imageURL = { uri: props.images[count]}
+
+    const leftPress = () => {
+        if (count > 0) setCount(count - 1);
```
Comment (ahmedsakr) [inline components/Restaurant.js:12]: separate line

File: components/Restaurant.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    const rightPress = () => {
+        if (count < images.length - 1) setCount(count + 1);
```
Comment (ahmedsakr) [inline components/Restaurant.js:16]: separate line

File: pages/SwipePage.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+import Header from '../components/Header';
+import Footer from '../components/Footer';
+import Restaurant from '../components/Restaurant';
+
+import Swiper from 'react-native-deck-swiper'
+import * as data from '../data/api-test-data.json';
+
+export default function SwipePage() {
+    function renderCard(card, index) {
+        let distanceKm = Math.round((data['businesses'][index].distance / 1000) * 10) / 10;
```
Comment (ahmedsakr) [inline pages/SwipePage.js:13]: hard to understand what's happening here

File: pages/SwipePage.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+                description={data['businesses'][index].categories[0].title}/>
+        )
+    };
+
+    return (
+        <View style={styles.container}>
+            <StatusBar style="light" />
+            <Header style={styles.header} />
+            <Swiper
+                cards={[...Array(data['businesses'].length).keys()]}
```
Comment (ahmedsakr) [inline pages/SwipePage.js:30]: why not `data['businesses'].keys()`?

**Review by ahmedsakr — APPROVED** [review]

File: pages/SwipePage.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+                renderCard={renderCard}
+                style={styles.test}
```
[the remark this answers is not in the document]
Comment (joshcampitelli) [reply pages/SwipePage.js]: 👌

File: components/Header.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    );
+}
+
+const styles = StyleSheet.create({
+    text: {
+        color: '#70EFDE',
+        marginTop: 30,
+        fontSize: 36,
+        fontFamily: 'Noteworthy-Bold',
+        position: 'absolute',
```
[the remark this answers is not in the document]
Comment (joshcampitelli) [reply components/Header.js]: Got it chief 🥇

File: components/Restaurant.js (hunk 1 above)
Comment (joshcampitelli) [reply components/Restaurant.js]: 😑

File: components/Restaurant.js (hunk 2 above)
Comment (joshcampitelli) [reply components/Restaurant.js]: 😧

File: pages/SwipePage.js (hunk 3 above)
Comment (joshcampitelli) [reply pages/SwipePage.js:14]: 🐵

File: pages/SwipePage.js (hunk 4 above)
Comment (joshcampitelli) [reply pages/SwipePage.js]: Ye

Comment (joshcampitelli) [reply pages/SwipePage.js]: `[...data['businesses'].keys()]`

**Review by MichaelCalabro — APPROVED** [review]

File: components/Footer.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { StyleSheet, Text, Image, View, TouchableOpacity } from 'react-native';
+
+const invite = require('../assets/happy.png');
```
Comment (ahmedsakr) [inline components/Footer.js:4]: use the `import` syntax to be consistent

`import invite from '../assets/happy.png`;`

same with others

**Review by ahmedsakr — APPROVED** [review]

**Pull request merged**